homeassistant/components/wittiot/coordinator.py

```diff
@@ -0,0 +1,48 @@
+"""The WittIOT integration coordinator."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+
+from aiohttp.client_exceptions import ClientConnectorError
+from wittiot import API
+from wittiot.errors import WittiotError
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN
+
+SCAN_INTERVAL = timedelta(seconds=60)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class WittiotDataUpdateCoordinator(DataUpdateCoordinator):
```

Comment:
```suggestion
class WittiotDataUpdateCoordinator(DataUpdateCoordinator[..]):
```
Please type the DataUpdateCoordinator with the return type of `_async_update_data`